package.json

```diff
@@ -2,13 +2,18 @@
   "name": "node-polyglot",
   "version": "2.3.0",
   "description": "Give your JavaScript the ability to speak many languages.",
-  "main": "index.js",
+  "main": "dist/polyglot.common-js.js",
+  "module": "dist/polyglot.es-modules.js",
   "scripts": {
-    "prepublish": "safe-publish-latest",
-    "pretest": "npm run --silent lint",
+    "build": "npm-run-all -p build:*",
+    "build:cjs": "rollup index.js --file dist/polyglot.common-js.js --format cjs",
+    "build:esm": "rollup index.js --file dist/polyglot.es-modules.js --format esm",
+    "prepublishOnly": "npm-run-all -s build safe-publish-latest",
```

Comment:
`prepublish` should be used; `prepublishOnly` requires a specific version range of npm.

Reply:
@ljharb I can definitely revert this. Something to be aware of:  `prepublish` is deprecated, and there's a good chance it will be removed in the future. if/when that happens there will suddenly be a specific version range of supported npm.

This is really only an issue for people that want to publish the module to npm, it won't affect general module consumers.

Reply:
That’s not the plan as i understand it, and if it is removed, we can address it then. It only affects package publishers, as you said, namely me.